apps/mobile/app/screens/Authenticated/ProfileScreen/components/UserProfileTasks.tsx

```diff
@@ -9,19 +9,22 @@ import { translate } from "../../../../i18n"
 import { typography, useAppTheme } from "../../../../theme"
 import { GLOBAL_STYLE as GS } from "../../../../../assets/ts/styles"
 import { observer } from "mobx-react-lite"
+import { useTimer } from "../../../../services/hooks/useTimer"
+import WorkedOnTaskHours from "../../../../components/WorkedDayHours"
 interface IUserProfileTasks {
 	profile: IUserProfile
 	content: ITaskFilter
 }
 const UserProfileTasks: FC<IUserProfileTasks> = observer(({ profile, content }) => {
 	const { colors, dark } = useAppTheme()
+	const { timerStatus } = useTimer()
 	const tasks = useMemo(() => {
-		let tasks = content.tasksFiltered
-		if (content.tab === "worked" && profile.activeUserTeamTask) {
-			tasks = tasks.filter((ts) => {
-				return ts.id !== profile.activeUserTeamTask?.id
-			})
-		}
+		const tasks = content.tasksFiltered
+		// if (content.tab === "worked" && profile.activeUserTeamTask) {
```

Comment:
@desperado1802 Lets remove commented code if code not needed.

Reply:
Done